src/screens/HomeStack/NftStack/components/nft-item-details/components/nft-item-details-attributes.tsx

```diff
@@ -6,35 +6,48 @@ import {Color} from '@app/colors';
 import {Spacer, Text, TextVariant} from '@app/components/ui';
 import {createTheme} from '@app/helpers';
 import {I18N} from '@app/i18n';
+import {NftAttribute} from '@app/models/nft';
 
 type Props = {
-  properties: Record<string, string> | null;
+  attributes?: NftAttribute[] | null;
 };
 
-export const NftItemDetailsAttributes = ({properties}: Props) => {
+export const NftItemDetailsAttributes = ({attributes}: Props) => {
+  if (!attributes) {
+    return null;
+  }
+
+  const valueConverter = (attr: NftAttribute) => {
+    if (attr.display_type === 'date') {
+      return new Date(Number(attr.value) * 1000).toLocaleDateString();
+    }
+
+    return attr.value;
+  };
+
   return (
     <>
       <Text variant={TextVariant.t12} i18n={I18N.nftDetailsAttributes} />
       <Spacer height={8} />
       <View style={styles.attributeListContainer}>
-        {properties
-          ? Object.entries(properties).map(([key, value]) => {
-              return (
-                <View key={key} style={styles.attributeContainer}>
-                  <View style={styles.attributeValueContainer}>
-                    <Text variant={TextVariant.t13}>{value}</Text>
-                    {/*<Text variant={TextVariant.t13}>*/}
-                    {/*  {value}*/}
-                    {/*  /!*{prop.frequency * 100}%*!/*/}
-                    {/*</Text>*/}
-                  </View>
-                  <Text variant={TextVariant.t15} color={Color.textBase2}>
-                    {key}
-                  </Text>
-                </View>
-              );
-            })
-          : null}
+        {attributes.map(attribute => {
+          return (
+            <View key={attribute.trait_type} style={styles.attributeContainer}>
+              <View style={styles.attributeValueContainer}>
+                <Text variant={TextVariant.t13}>
+                  {valueConverter(attribute)}
+                </Text>
+                {/*<Text variant={TextVariant.t13}>*/}
```

Comment:
Unused code

Reply:
Yes. This part will be uncomment, when server will return frequency of nft